docs/source/common/test_set.rst

```diff
@@ -101,3 +129,21 @@ you can also pass in an :doc:`datamodules <../extensions/datamodules>` that have
 
     # test (pass in datamodule)
     trainer.test(datamodule=dm)
+
+----------
+
+**********
+Validation
+**********
+
+Lightning allows the user to validate their models with any compatible val dataloaders. This can be done before/after training.
```

Comment:
```suggestion
Lightning allows the user to validate their models with any compatible ``val dataloaders``. This can be done before/after training.
```